components/cam/bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -1389,6 +1391,10 @@
 <statefreq> 480 </statefreq>
 <integration> "explicit" </integration>
 
+<cubed_sphere_map> 0 </cubed_sphere_map>
```

Comment:
This is probably related to the same issue with namelist.

Reply:
Per our phone conversation, I removed this and added a section to the "namelist_mod.F90" in the HOMME code that handles the requirement that cubed_sphere_map=2 for fv_nphys>0.